extensions/objects/templates/svg-filename-editor.component.html

```diff
@@ -91,6 +117,11 @@
                 tooltip-placement="left" uib-tooltip="Pie Chart" ng-disabled="!$ctrl.isPieChartEnabled()">
           <i class="fas fa-chart-pie pie-chart"></i>
         </button>
+        <button class="icon-tool toolbar-button thin-button protractor-test-create-svgupload"
+                ng-class="{'disabled' : !$ctrl.isSvgUploadEnabled()}" ng-click="$ctrl.uploadSvgFile()"
+                tooltip-placement="right" uib-tooltip="Upload SVG" ng-disabled="!$ctrl.isSvgUploadEnabled()">
+          <i class="far fa-file-image file-upload"></i>
```

Comment:
Hm, can you point me to where `far` is defined?

Reply:
```far``` is a part of font-awesome styles.